.github/workflows/main.yml

```diff
@@ -107,10 +107,9 @@ jobs:
       - name: download HDF5 for Windows
         run: |
           wget --quiet \
-            https://support.hdfgroup.org/releases/hdf5/v1_14/v1_14_1/downloads/hdf5-1.14.1-2-Std-win10_64-vs17-clang.zip
-          unzip hdf5-1.14.1-2-Std-win10_64-vs17-clang.zip
+            https://support.hdfgroup.org/releases/hdf5/v1_14/v1_14_5/downloads/hdf5-1.14.5-win-vs2022_cl.msi
```

Comment:
Thanks for updating to current. I had to go find a 1.14 release that was compatible enough with existing logic here that I didn't have to change much because I don't follow the Windows command line logic too well.